skype/skype-ps/skype/Get-CsTeamsAppSetupPolicy.md

```diff
@@ -42,8 +42,45 @@ Apps are pinned to the app bar. This is the bar on the side of the Teams desktop
 ## EXAMPLES
 
 ### Example 1
-Intentionally not provided
 
+Get the Global Teams App Setup Policy.
```

Comment:
please move the description of the example below the code (Leaving a carrier return too)

Reply:
thanks for your reminder, fixed it.